contracts/UserYieldFarming.sol

```diff
@@ -1441,8 +1452,15 @@ contract MasterChef is Ownable {
         
         
         if (_amount > 0) {
-            pool.lpToken.safeTransferFrom(address(msg.sender), address(this), _amount);
-            user.amount = user.amount.add(_amount);
+            pool.lpToken.safeTransferFrom(msg.sender, address(this), _amount);
+            uint256 userPct = ((_amount * 1.3E18) / 100E18);
+            uint256 newAmt = (_amount - userPct);
```

Comment:
you should'nt remove 1.3% from all deposit, it should only have effect on external pool.